steps/gcp-step-cloud-run-deploy/spec.schema.json

```diff
@@ -0,0 +1,131 @@
+
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "credentials": {
+        "type":"object",
+        "description":"Credentials to use when running this step.",
+        "properties": {
+            "connection": {
+                "type": "object",
+                "x-relay-connectionType": "gcp",
+                "description": "A Relay Google Cloud Platform (GCP) connection to use",
+                "properties": {
+                  "service-account.json": {
+                    "type": "string",
+                    "description": "A base64-encoded JSON service account key to use for deploying the revision."
+                  }
+                },
+                "required": [
+                  "service-account.json"
+                ]
+              }
+        },
+        "required":[
+            "connection"
+        ]
+    },
+    "project": {
+        "type":"string",
+        "description":"The identifier of the Google Cloud project to deploy to."
+    },
+    "service": {
+        "type":"string",
+        "description":"The name of the service to deploy to."
+    },
+    "region": {
+        "type":"string",
+        "description":"The Google Cloud region to deploy to."
+    },
+    "image": {
+        "type":"object",
+        "description":"The Docker image to deploy.",
+        "properties": {
+            "repository": {
+                "type": "string",
+                "description": "The name of the repository containing the Docker image."
+            },
+            "tag": {
+                "type": "string",
+                "description":"The tag to deploy.",
+                "default":"latest"
+            }
+        },
+        "required":[
+            "repository"
+        ]
+    },
+    "serviceAccount": {
+        "type":"object",
+        "description":"The service account to use when executing the service",
+        "properties": {
+            "email": {
+                "type": "string",
+                "description": "The email address of the service account"
+            }
+        },
+        "default": "Project default"
```

Comment:
Don't do this, just omit `"default"` if it isn't a literal default value. It's not supposed to be prose.